lib/pages/chat/chat.dart

```diff
@@ -2015,7 +2026,10 @@ class ChatController extends State<Chat>
   Widget build(BuildContext context) {
     return MouseRegion(
       onHover: (_) => _resetLocationPath(),
-      child: ChatView(this, key: widget.key),
+      child: GestureDetector(
+        onTap: requestNotificationPermission,
```

Comment:
Why not use `onEnter ` of `MouseRegion`.

Reply:
if we use onEnter it won't support click. Also with gesture detector we will be able to ask notification if the user enters a room but for example just waits for a message